vendor/engines/projects/app/models/projects/project.rb

```diff
@@ -5,12 +5,15 @@ class Project < ActiveRecord::Base
     include ActiveModel::ForbiddenAttributesProtection
 
     belongs_to :facility, foreign_key: :facility_id
+    has_many :order_details, inverse_of: :project
 
     validates :facility_id, presence: true
     validates :name,
               presence: true,
               uniqueness: { case_sensitive: false, scope: :facility_id }
 
+    scope :active, conditions: { active: true }
```

Comment:
Does 3.2 allow `scope :active, -> { where(active: true) }`? I feel like we should be using a more modern syntax.